src/main/java/sx/blah/discord/handle/impl/obj/PrivateChannel.java

```diff
@@ -167,4 +167,9 @@ public String toString() {
 	public IPrivateChannel copy() {
 		return new PrivateChannel(client, recipient, id);
 	}
+
+	@Override
+	public boolean isDeleted() {
+		return getClient().getChannelByID(id) != this;
```

Comment:
This should just return false. There is no case where a private channel should be "deleted" in the case of how this api handles private channels.